### ISSUE TITLE
debt in real time scraped

### ISSUE THREAD

**Issue by jmmartinnu2** [OpenBB-finance/OpenBBTerminal#2269]
**What's the problem of not having this feature?**
from here you can see the national debt of any country in real time... it's interesting if you insert it too
![scraped-debt](https://user-images.githubusercontent.com/93231115/183395964-e778c9b0-3d18-49c0-bdf9-1653e9ea7334.png)

**Describe the solution you would like**
A description of what you think would be the best solution for this. From user inputs, to an output.

**Describe alternatives you've considered**
If there are other alternatives (worth considering) to your solution, list them here. 

**Additional information**
Add any other information or screenshots about the feature request here.

**Comment by JerBouma** [comment]
Why is this feature relevant for you to include? E.g. what do you infer from it that helps in your investment decisions?

**Comment by jmmartinnu2** [comment]
It is more visual information in case someone is curious about the level of debt, like a command that opens the web directly. Similar to when you open finviz heatmap. clear debt that affects the investment decision! example can be a determining factor if, for example, you are thinking of investing in emerging countries and first of all you want to see their level of indebtedness

**Comment by colin99d** [comment]
@JerBouma if this feature was added, where would you want to see it?

**Comment by JerBouma** [comment]
Within `economy`, over here:
![image](https://user-images.githubusercontent.com/46355364/183892668-44ffea20-2829-411c-a846-f0fec32c6607.png)

**Comment by colin99d** [comment]
I was going to scrape the data.

**Comment by JerBouma** [comment]
Oh that sounds good then!

**Comment by colin99d** [comment]
It looks like that website will not be possible to scrape. Would this one be useful: https://commodity.com/data/debt-clock/. I am not sure how often they update it.

**Comment by JerBouma** [comment]
Yeah that source looks good, their data comes from sources that tend to update monthly. While you are at it, could you include in the table the following

- Country name
- Total debt
- Population
- Debt per capita (total debt / population)

Also good to note that it is all in U.S. dollars to prevent confusion when you look at the Debt of the Netherlands for example.
